lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -179,25 +183,40 @@ class OrbitalGateway < Gateway
       USE_ORDER_ID         = 'O' #  Use OrderID field
       USE_COMMENTS         = 'D' #  Use Comments field
 
-      SENSITIVE_FIELDS = [:account_num, :cc_account_num]
+      SENSITIVE_FIELDS = %i[account_num cc_account_num]
+
+      # Bank account types to be used for check processing
+      ACCOUNT_TYPE = {
+        'savings' => 'S',
+        'checking' => 'C'
+      }
 
       def initialize(options = {})
         requires!(options, :merchant_id)
         requires!(options, :login, :password) unless options[:ip_authentication]
         super
+        @options[:merchant_id] = @options[:merchant_id].to_s
```

Comment:
I wonder why this has been added in upstream 🤔

Reply:
This is specific PR on that :)
https://github.com/activemerchant/active_merchant/commit/e7aad1c72559aaef84634d7b8a43b2a8264442ed

Reply:
ooh I see they wanted to support numerical merchant_id so they overriden this